### DISCUSSION TITLE
issue/bug: incomplete inventories

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#56] [Shipped - Live on production]
At the moment I have only these requiremetns for incomplete inventories:
![image](https://user-images.githubusercontent.com/117633901/214231185-c5562da4-fb7f-470a-a91f-63ead366d8d6.png)

If I look in the inventory, and filter Device Location column on Is empty I get these 5 items with empty location (all other filters off):
![image](https://user-images.githubusercontent.com/117633901/214231289-357d5990-e662-4916-8c95-8e6be673a1c8.png)

But on the dashboard view it says 3 incomplete inventories:
![image](https://user-images.githubusercontent.com/117633901/214231355-d2427d48-935a-4ffd-8d37-57bf04544daf.png)

Is the dasboard view not taking the location as a missing thing or..?

**Comment by YvesR** [comment]
We are investigating this issue and let you know, seems to be a glitch in the counter caching.

**Reply by YvesR** [reply]
There was a logic bug that has been fixed and rolled out, working again as expected.
